### PR TITLE
chore: Update CI to assume github actions

### PR DESCRIPTION
**Pull request by Marwes**

### CODE REVIEW

File: src/parser/range.rs [hunk 1]
```diff
@@ -85,7 +85,6 @@ parser! {
     /// assert!(parser.parse("!").is_err());
     /// # }
     /// ```
-    #[inline]
```
Comment (klensy) [inline src/parser/range.rs:88]: Looks unrelated?

Comment (Marwes) [reply src/parser/range.rs:88]: It was a new warning about duplicate inline so I just removed it as I went.
